pkg/operator/ceph/disruption/nodedrain/reconcile.go

```diff
@@ -151,16 +160,46 @@ func (r *ReconcileNode) reconcile(request reconcile.Request) (reconcile.Result,
 		// a new object is going to be created
 		if deploy.ObjectMeta.CreationTimestamp.IsZero() {
 			deploy.Spec.Selector = &metav1.LabelSelector{
-				MatchLabels: deploymentLabels,
+				MatchLabels: selectorLabels,
+			}
+		}
+
+		ownerReferences := []metav1.OwnerReference{}
+		// get the operator Deployment to use as an owner refreence
+		operatorPodKey := types.NamespacedName{Name: os.Getenv(k8sutil.PodNameEnvVar), Namespace: r.context.OperatorNamespace}
+		operatorDeployment, err := getDeploymentForPod(r.client, operatorPodKey)
+		if err != nil {
+			logger.Errorf("could not find rook operator deployment for pod %+v: %+v", operatorPodKey, err)
+		} else {
+
+			operatorDeployment := operatorDeployment
+			controllerBool := true
```

Comment:
When should `Controller` be true? I'm not clear on this arg.

Reply:
"If true, this reference points to the managing controller."